### DISCUSSION TITLE
Do you plan to develop SeaTools and add new features (if possible)?

### DISCUSSION THREAD

**Discussion by fedotovartuom76** [Seagate/ToolBin#20] [Q&A] [answered]
Hi all. 
Now SeaTools has less functionality than SeaChest and it has not received new features for a long time. Is it possible to expand SeaTools functionality in principle? Or other features just impossible to implement there?

**Comment by Rungthida** [comment, marked as answer]
Hi @fedotovartuom76,
Thanks for the question.
Seatools has always been a simple end user tool targeted at helping test a drive and erase a drive. SeaChest is meant to do “everything”.
If there is a specific feature you think would be valuable to add to SeaTools, we will look into it.
We have been working on a SeaTools for Windows replacement for a while now and it will be released soon. We are wrapping up some last issues as we've been testing it.
This will combine SeaTools for Windows and SeaTools SSD into one tool.

**Comment by LocoLati** [comment]
IMHO I like the idea of a "One For All" tool, a single tool for all that useful features.  Maybe giving the GUI a choice for Simple Tasks or switching to an Advanced Mode or a (may be password protected) Expert Mode.  I guess that the coding for all those nice features is already there, isnt it?  
;-)

**Reply by vonericsen** [reply]
I've considered this idea and will bring it up to our team for discussion. We've internally talked about keeping Seatools as-is and creating the more advanced tool as a Seachest GUI too. I'm not sure if/when this will get worked on though.
The "talk to the drive" parts are mostly done, but we do spend a lot of time figuring out the best way to take user input and pass it to these functions so that it is clear to the users. We have more things planned for upcoming versions of SeaTools, but not quite as much as is in SeaChest CLI tools right now.
